vllm/core/block/cpu_gpu_block_allocator.py

```diff
@@ -190,10 +190,16 @@ def clear_copy_on_writes(self) -> Dict[int, List[int]]:
         device = Device.GPU
         return self._allocators[device].clear_copy_on_writes()
 
-    def mark_blocks_as_computed(self) -> None:
+    def mark_blocks_as_accessed(self, block_ids: List[int],
+                                now: float) -> None:
         # Prefix caching only supported on GPU.
         device = Device.GPU
-        return self._allocators[device].mark_blocks_as_computed()
+        return self._allocators[device].mark_blocks_as_accessed(block_ids, now)
+
+    def mark_blocks_as_computed(self, seq_block_ids: List[int]) -> None:
+        # Prefix caching only supported on GPU.
+        device = Device.GPU
+        return self._allocators[device].mark_blocks_as_computed(seq_block_ids)
```

Comment:
can we validate that the block ids are on GPU? otherwise the error message will be pretty confusing

Reply:
I tend to the logic of check whether block has content_hash for judging whether it is computed, so we no longer need this function in cpu_gpu_block_allocator.